src/app/pages/TopicPage/HierarchicalGrid/index.tsx

```diff
@@ -0,0 +1,100 @@
+/** @jsx jsx */
+
+import { css, jsx, Theme } from '@emotion/react';
+import Promo from '../../../legacy/components/Promo';
+import { DESKTOP, TABLET, MOBILE, SMALL } from './dataStructures';
+
+type Summary = {
```

Comment:
Double check which are required and which are optional

Reply:
you got this